packages/IMAPClient-Core.package/ICCommand.class/class/fetchFlagsForUIDFrom.to..st

```diff
@@ -1,4 +1,4 @@
 commands
-fetchFlagsWithUIDFrom: aBeginUID to: anEndUID
+fetchFlagsForUIDFrom: aBeginUID to: anEndUID
```

Comment:
Please make clear that this method works on a UID range.
Use `fetchFlagsFromUIDsFrom: to:` or `fetchFlagsFromUIDRangeFrom`